pkg/apiserver/resource_manager/service.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2023 PingCAP, Inc. Licensed under Apache-2.0.
+
+package resourcemanager
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"go.uber.org/fx"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/user"
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/utils"
+	"github.com/pingcap/tidb-dashboard/pkg/tidb"
+	"github.com/pingcap/tidb-dashboard/util/featureflag"
+	"github.com/pingcap/tidb-dashboard/util/rest"
+)
+
+type ServiceParams struct {
+	fx.In
+	TiDBClient *tidb.Client
+}
+
+type Service struct {
+	FeatureResourceManager *featureflag.FeatureFlag
+
+	params ServiceParams
+}
+
+func newService(p ServiceParams, ff *featureflag.Registry) *Service {
+	return &Service{params: p, FeatureResourceManager: ff.Register("resource_manager", ">= 7.0.0")}
+}
+
+func registerRouter(r *gin.RouterGroup, auth *user.AuthService, s *Service) {
+	endpoint := r.Group("/resource-manager")
+	endpoint.Use(
+		auth.MWAuthRequired(),
+		s.FeatureResourceManager.VersionGuard(),
+		utils.MWConnectTiDB(s.params.TiDBClient),
+	)
+	{
+		endpoint.GET("/information", s.GetInformation)
+		endpoint.GET("/config", s.GetConfig)
+		endpoint.GET("/calibrate", s.GetCalibrate)
+	}
+}
+
+// @Summary Get Resource Control enable config
+// @Router /resource-manager/config [get]
+// @Security JwtAuth
+// @Success 200 {string} enable
+// @Failure 401 {object} rest.ErrorResponse
+// @Failure 500 {object} rest.ErrorResponse
+func (s *Service) GetConfig(c *gin.Context) {
+	db := utils.GetTiDBConnection(c)
+	var enable string
+	err := db.Raw("SELECT @@GLOBAL.tidb_enable_resource_control as tidb_enable_resource_control").Find(&enable).Error
+	if err != nil {
+		rest.Error(c, err)
+		return
+	}
+	c.JSON(http.StatusOK, enable)
+}
+
+type TableRowDef struct {
+	Name      string `json:"name" gorm:"column:NAME"`
+	RuPerSec  string `json:"ru_per_sec" gorm:"column:RU_PER_SEC"`
+	Priority  string `json:"priority" gorm:"column:PRIORITY"`
+	Burstable string `json:"burstable" gorm:"column:BURSTABLE"`
+}
+
+// @Summary Get Information of Resource Groups
+// @Router /resource-manager/information [get]
+// @Security JwtAuth
+// @Success 200 {object} TableRowDef
+// @Failure 401 {object} rest.ErrorResponse
+// @Failure 500 {object} rest.ErrorResponse
+func (s *Service) GetInformation(c *gin.Context) {
+	db := utils.GetTiDBConnection(c)
+	cfg := &TableRowDef{}
+	err := db.Table("INFORMATION_SCHEMA.RESOURCE_GROUPS").Find(cfg).Error
+	if err != nil {
+		rest.Error(c, err)
+		return
+	}
+	c.JSON(http.StatusOK, cfg)
```

Comment:
In frontend, we expect the cfg as an array, likes:

```
[ 
  {name: '', ru_per_sec: '', priority: '', burstable: ''}, 
  {name: '', ru_per_sec: '', priority: '', burstable: ''},
 ...]
```

Reply:
done.